apps/cowswap-frontend/src/common/updaters/BridgingEnabledUpdater.ts

```diff
@@ -14,11 +14,13 @@ export function BridgingEnabledUpdater(): null {
   const tradeTypeInfo = useTradeTypeInfo()
   const accountType = useAccountType()
   const setIsBridgingEnabled = useSetIsBridgingEnabled()
+  const { isBridgingEnabled: isBridgingEnabledFlag } = useFeatureFlags()
```

Comment:
Due to sync with develop, this is effectively adding the feature flag back to the code, no?